dapr/proto/operator/v1/operator.proto

```diff
@@ -100,3 +104,34 @@ message ListSubscriptionsRequest {
   string podName = 1;
   string namespace = 2;
 }
+
+// GetHTTPEndpointRequest is the request to get an http endpoint configuration.
+message GetHTTPEndpointRequest {
+  string name = 1;
+  string namespace = 2;
+}
+
+// GetHTTPEndpointResponse includes the requested http endpoint configuration.
+message GetHTTPEndpointResponse {
+  bytes http_endpoint = 1;
+}
+
+// ListHTTPEndpointsResponse includes the list of available http endpoint configurations.
+message ListHTTPEndpointsResponse {
+  repeated bytes http_endpoints = 1;
+}
+
+message ListHTTPEndpointsRequest {
+  string namespace = 2;
+}
+
+// HTTPEndpointsUpdateRequest is the request to get updates about new http endpoints for a given namespace.
+message HTTPEndpointsUpdateRequest {
+  string namespace = 1;
+  string podName = 2;
```

Comment:
Protobuf uses snake_case by convention:

```suggestion
  string pod_name = 2;
```

Reply:
I see now that other messages use `podName`. We may need to change that, if not now in the future, (it should be backwards-compatible anyways), but let's start with the new messages at least :)